src/ducks/classifications.js

```diff
@@ -186,38 +303,9 @@ const submitClassification = () => {
         height: innerHeight,
       },
       'metadata.subject_dimensions': subject_dimensions || [],
-    })
-      .save()
-
-      //Successful save: reset everything, then get the next Subject.
-      .then(() => {
-        if (user) {
-          localStorage.removeItem(`${user.id}.classificationID`);
-        }
-
-        try {  //Fix: IE11 doesn't know what to do with Split.classificationCreated()
-          Split.classificationCreated(classification);
-        } catch (err) { console.error('Split.classificationCreated() error: ', err); }
-
-        //Reset values in preparation for the next Subject.
-        console.info('ducks/classifications.js submitClassification() success');
-        dispatch({ type: SUBMIT_CLASSIFICATION_SUCCESS });
-        dispatch(fetchSubject());  //Note: fetching a Subject will also reset Annotations, reset Previous Annotations, and create an empty Classification.
-        dispatch(resetView());
-      })
-
-      //Unsuccessful save
-      .catch((err) => {
-        console.error('ducks/classifications.js submitClassification() error: ', err);
-        Rollbar && Rollbar.error &&
-        Rollbar.error('ducks/classifications.js submitClassification() error: ', err);
-        alert('ERROR: Could not submit Classification');  //TODO: better messaging
-        dispatch({ type: SUBMIT_CLASSIFICATION_ERROR });
-      });
-    //----------------
-
-    const { workflow, subjects } = classification.links;
-    dispatch(addAlreadySeen(workflow, subjects));
+    });
+    queueClassification(classification, user);
+    saveAllQueuedClassifications(dispatch, getState);
```

Comment:
MINOR: In `queueClassification(classification, user)` you pass in the user object. In `savedAllQueuedClassifications(dispatch, getState)` you pass in `getState` to immediately call `getState().login.user` in that function. Maybe go with `user` for consistency?

Reply:
That makes a lot of sense. I'm on it.